Source/WebInspectorUI/UserInterface/Views/SpreadsheetRuleHeaderField.js

```diff
@@ -110,9 +111,9 @@ WI.SpreadsheetSelectorField = class SpreadsheetSelectorField extends WI.Object
         if (document.activeElement === this._element)
             return;
 
-        if (this._delegate && typeof this._delegate.spreadsheetSelectorFieldDidCommit === "function") {
+        if (typeof this._delegate?.spreadsheetRuleHeaderFieldDidCommit === "function") {
```

Comment:
NIT: you can also drop the `typeof` part too.  we don't support the possibility that a property with the same name as a delegate method be anything other than a function